car-station-service/src/main/java/com/example/carstationservice/mapper/CarMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.example.carstationservice.mapper;
+
+
+import com.example.carstationservice.dto.request.CarRequest;
+import com.example.carstationservice.dto.response.CarResponse;
+import com.example.carstationservice.model.Car;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface CarMapper {
+    CarMapper INSTANCE = Mappers.getMapper(CarMapper.class);
```

Comment:
As far as I remember, you can try injecting mapper into controller/service constructor without creating instance of it

Reply:
You can mark the mapper with an annotation: Mapper(componentModel = "sping")